champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/bow/MarkedForDeath.java

```diff
@@ -76,7 +77,7 @@ public void onHit(Player damager, LivingEntity target, int level) {
 
     @Override
     public void displayTrail(Location location) {
-        Particle.REDSTONE.builder().location(location).color(5, 5, 5).count(3).extra(0).receivers(60, true).spawn();
+        location.getWorld().spawnParticle(Particle.SPELL_MOB, location, 1, 0.1, 0.1, 0.1, 0);
```

Comment:
Any reason you didnt use the builder here? I believe it gives better control over some things e.g. view distance to see the particle

Reply:
I did not know that the particle builder supported particle displacement/other particles besides dust and I'm also not sure it works to do the cheaty little hack I did for the MOB_SPELL where the delta values turn to rgb values when the particle number is set to 0. I'll try it and see if it works with the specifications I need

Reply:
fixed